packetbeat/etc/fields.yml

```diff
@@ -1,39 +1,14 @@
-version: 1.0
-
 defaults:
   type: keyword
   required: false
   index: true
-  doc_values: true
   ignore_above: 1024
 
-flows_env:
-  type: group
-  description: >
-    These fields contain data about the environment in which the flow data was
-    captured.
-  fields:
-    - name: beat.name
-      description: >
-        Name of the Beat sending the events. If the shipper name is set
-        in the configuration file, then that value is used. If it is not set,
-        the hostname is used.
-
-    - name: beat.hostname
-      description: >
-        The hostname as returned by the operating system on which the Beat is
-        running.
-
-    - name: tags
-      description: >
-        Arbitrary tags that can be set per Beat and per transaction
-        type.
-
-trans_env:
+env:
```

Comment:
Yep, not important, was more thinking to have it potentially the same in the future across all beats (not sure if that is already the case).